recipes-asteroid/asteroid-package-feed/asteroid-package-feed.bb

```diff
@@ -0,0 +1,30 @@
+SUMMARY = "Build community packages and create package index from the community packages and rootfs pacakges."
+LICENSE = "MIT"
+
+INHIBIT_DEFAULT_DEPS = "1"
+PACKAGES = ""
+
+# A set of packags to build for the package feed bu not include in rootfs.
```

Comment:
but that are not included*

Reply:
ow, wow. Looks almost as if my keyboard is broken :smile:
Sorry for not checking this...